src/components/Header/index.js

```diff
@@ -1,97 +1,108 @@
-import React, { useState } from 'react'
+import React, { Fragment, useState, useEffect } from 'react'
 
 import styled from 'styled-components'
 import Flex from 'styled-flex-component'
 import { Padding } from 'styled-components-spacing'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
 
+import navLinks from './navLinks'
 import { Row, Col, Grid } from '../grid'
 import Hamburger from './Hamburger'
 import Overlay from './Overlay'
 import TopNav from './TopNav'
 import SideNav from './SideNav'
 
-const FixedHeightFlex = styled(Flex)`
-  height: 84px;
+const StyledPadding = styled(Padding)`
+  display: flex;
+  height: ${remcalc(120)};
+  align-items: center;
 `
 
-const navLinks = [
-  {
-    label: 'Services',
-    dropdownItems: [
-      {
-        label: 'Engineering',
-        to: '/engineering/'
-      },
-      {
-        label: 'Design',
-        to: '/design/'
-      },
-      {
-        label: 'Training',
-        to: '/training/'
-      }
-    ]
-  },
-  {
-    label: 'Our work',
-    to: '/our-work/'
-  },
-  {
-    label: 'Blog',
-    href: 'https://medium.com/yld-engineering-blog/'
-  },
-  {
-    label: 'About',
-    dropdownItems: [
-      {
-        label: 'Our team',
-        to: '/about-us/'
-      },
-      {
-        label: 'Contact',
-        to: '/contact/'
-      }
-    ]
-  },
-  {
-    label: 'Join us',
-    to: '/join-us/'
-  }
-]
+const StyledGrid = styled(Grid)`
+  position: fixed;
+  height: ${remcalc(84)};
+  background: ${props =>
+    props.theme.colors[props.isSpecialityPage ? 'blueBg' : 'white']};
+  width: 100%;
+  max-width: unset;
+  z-index: 100;
+
+  ${breakpoint('smallTablet')`
```

Comment:
Nice for the Grid component. Was able to remove  the whole smallTablet breakpoint 👍 
However I still need the desktop one because at this point a max-width is 1100px and without 
`display: flex;` & `justify-content: center;` it's aligned on the left.
The whole ticket is so tricky because of all these 5 breakpoints + resizing of the logos (also different logos) + sometimes the TopNav has the service link or not and hamburger can replace dropdown. It

Reply:
Yes, but by using the `Grid` component, they shouldn't be aligned on the left. The one benefit of this component is that you actually don't need to concern yourself with all of these breakpoints because this component should abstract away that complexity.

So you shouldn't need to add extra code on top of `Grid`, in order to make the logos conform to our grid system :thinking: